extension/src/setup/commands/index.ts

```diff
@@ -275,25 +272,21 @@ const confirmAndRemove = async (
     return
   }
 
-  try {
-    await internalCommands.executeCommand(
-      AvailableCommands.REMOTE,
-      dvcRoot,
-      SubCommand.REMOVE,
-      Flag.PROJECT,
-      remote
-    )
-  } catch {}
```

Comment:
I don't think we don't need these try/catch blocks since `executeSafeProcess` already catches errors.

Reply:
is there a test for this?

Reply:
I can add a test to ensure `remote` catches errors.